hack/update-generated-crd-code.sh

```diff
@@ -28,8 +30,8 @@ if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
   exit 1
 fi
 
-if [[ ! -d "${GOPATH}/src/sigs.k8s.io/controller-tools" ]]; then
-  echo "sigs.k8s.io/controller-tools missing from GOPATH"
+if ! command -v controller-gen > /dev/null; then
+  echo "controller-gen is missing"
```

Comment:
should we also say how to get it?

Reply:
this is really only intended to be run inside the build container, which already includes it, so I don't think this matters too much